### ISSUE TITLE
Do we really need the `FixScaleCrop` in the `transform_val`?

### ISSUE THREAD

**Issue by LarT2P** [jfzhang95/pytorch-deeplab-xception#76]
I'm confused about it.

For validation sets, especially here, it's more like a test set. do we need to cut the picture? 

Do you think it would be more appropriate to just use the resize operation? 
After all, the clipping operation discards too much data. is such a measurement reliable? 

If I have some mistake, please point it out.

https://github.com/jfzhang95/pytorch-deeplab-xception/blob/c8ff02c6eeb2b774ad5a25557a60ee48e04635c6/dataloaders/datasets/pascal.py#L94

```python
    def transform_val(self, sample):

        composed_transforms = transforms.Compose([
            tr.FixScaleCrop(crop_size=self.args.crop_size),
            tr.Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
            tr.ToTensor()])

        return composed_transforms(sample)
```

**Comment by laplaceson** [comment]
Same question ,discarding some information. 
Also thanks for the code!

**Comment by LarT2P** [comment]
> Same question ,discarding some information.
> Also thanks for the code!

:handshake: I read some other code, they also do it like this...

Maybe, it's okay.

like this:

https://github.com/kevin-ssy/FishNet/blob/ba57a0718deeedab625534506f53d413e2d059cf/main.py#L133
